docs/scos/dev/feature-walkthroughs/202204.0/configurable-product-feature-walkthrough/configurable-product-flow-chart.md

```diff
@@ -0,0 +1,198 @@
+---
+title: Configurable Product flow chart
+last_updated: Jun 2, 2022
+description:
+template: concept-topic-template
+---
+
+This document describes the flow chart that demonstrates the configuration process of the configurable product.
+
+## Flow chart
+
+The following flow chart represents the configuration process flow of the configurable product:
+
+![configuration-flow](https://confluence-connect.gliffy.net/embed/image/49194e23-84dc-4dc8-8ce2-29fa244522b3.png?utm_medium=live&utm_source=custom)
+
+## Product configuration data flow on the product details page
+
+When configuration starts on Yves, from the product details page (PDP), the product configuration data flow consists of the following phases.
+
+### Phase 1
+
+- We are on a PDP—the page with the product configuration.
+- The product configuration can be already complete or use not complete pre-configuration defined by the shop owner.
+- The product configuration can be taken from two sources:
+  - A session for complete configuration.
+  - Storage (Redis) for the pre-configuration.
+
+The following table shows the configuration data that is stored in the Session and Storage.
+
+| PARAMETER | VALUE | COMMENTS  |
+|---|---|---|
+| `ProductConfigurationInstance.isComplete` | `true` or `false` | Sensitive data. |
+| `ProductConfigurationInstance.displayDescription` | Some text of JSON blob—fore example, `["color"=>"red", "weight"=> 100]` |   |
+| `ProductConfigurationInstance.configuratorKey` | `dateTime` |   |
+| `ProductConfigurationInstance.configuration` |  `["color"=>"red", "weight"=> 100]` | Sensitive data. |
+
+We will generate a URL that points to the gateway page with the following parameters:
```

Comment:
Can it be like
```suggestion
Generate a URL that points to the gateway page with the following parameters:
```
or

```suggestion
You generate a URL that points to the gateway page with the following parameters:
```

Reply:
First? But what is our policy on writing these texts? Do we address the reader directly (you)? If we do so then we should do it everywhere.

Reply:
Yes, we address the reader directly and avoid using "we" if we do not talk on behalf of Spryker. 
And yes, we have to do it everywhere, but since the document is too technical, I'm not sure where exactly to make changes.

Reply:
The system or framework is generating the back URL.